sdcm/cluster.py

```diff
@@ -2165,29 +2166,11 @@ def config_scylla_manager(self, mgmt_port, db_hosts):
         :param db_hosts:
         :return:
         """
-        # only support for centos
-        self.log.debug('Install scylla-manager')
-        rsa_id_dst = '/tmp/scylla-test'
-        mgmt_conf_tmp = '/tmp/scylla-manager.yaml'
-        mgmt_conf_dst = '/etc/scylla-manager/scylla-manager.yaml'
-
-        mgmt_conf = {'http': '0.0.0.0:{}'.format(mgmt_port),
-                     'database':
-                         {'hosts': db_hosts,
-                          'timeout': '5s'},
-                     'ssh':
-                         {'user': self.ssh_login_info['user'],
-                          'identity_file': rsa_id_dst}
-                     }
-        (_, conf_file) = tempfile.mkstemp(dir='/tmp')
-        with open(conf_file, 'w') as fd:
-            yaml.dump(mgmt_conf, fd, default_flow_style=False)
-        self.remoter.send_files(src=conf_file, dst=mgmt_conf_tmp)  # pylint: disable=not-callable
-        self.remoter.run('sudo cp {} {}'.format(mgmt_conf_tmp, mgmt_conf_dst))
-        if self.is_docker():
-            self.remoter.run('sudo supervisorctl start scylla-manager')
-        else:
-            self.remoter.run('sudo systemctl restart scylla-manager.service')
+        mgmt_conf_dst = "/etc/scylla-manager/scylla-manager.yaml"
+        with self.parent_cluster.remote_scylla_yaml(path=mgmt_conf_dst) as scylla_yml:
```

Comment:
I think using `remote_scylla_yaml` directly might be a bit confusing (since it's not `scylla.yaml` that is being edited)
@enaydanov what do you think ?